packages/ui/src/elements/BulkUpload/FileSidebar/index.tsx

```diff
@@ -157,7 +156,8 @@ export function FileSidebar() {
                   >
                     <Thumbnail
                       className={`${baseClass}__thumbnail`}
-                      fileSrc={isImage(currentFile.type) ? thumbnailUrls[index] : undefined}
```

Comment:
@paulpopus That was the reason why video thumbnail wasn't display before we saved the media.